ReactCommon/hermes/inspector/tools/run_msggen

```diff
@@ -12,7 +12,7 @@ FBSOURCE=$(hg root)
 CLANG_FORMAT="${FBSOURCE}/tools/third-party/clang-format/clang-format"
 SIGNEDSOURCE="${FBSOURCE}/tools/signedsource"
 
-cd "${DIR}/msggen"
+cd "${DIR}/../../../../packages/hermes-inspector-msggen"
```

Comment:
Can we `pushd` and `popd` here as the paths are so distant?

Reply:
Sure, I've just pushed a commit updating this